xml/System.Threading.Tasks/ValueTask`1.xml

```diff
@@ -751,7 +751,13 @@ If the <xref:System.Threading.Tasks.ValueTask`1> is backed by a result or by a <
       <Docs>
         <summary>Gets a <see cref="T:System.Threading.Tasks.ValueTask`1" /> that may be used at any point in the future.</summary>
         <returns>A task object for future use.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+## Remarks
+
+Because a <xref:System.Threading.Tasks.ValueTask%601> can be backed by a pooled <xref:System.Threading.Tasks.Sources.IValueTaskSource%601>, it is not safe to await an arbitrary <xref:System.Threading.Tasks.ValueTask%601> multiple times. You can use the <xref:System.Threading.Tasks.ValueTask%601.Preserve> method to convert this <xref:System.Threading.Tasks.ValueTask%601> into the instance backed by a regular <xref:System.Threading.Tasks.Task%601> that is safe to await multiple times. This method is similar to <xref:System.Threading.Tasks.ValueTask%601.AsTask>, but it returns the same <xref:System.Threading.Tasks.ValueTask%601> instance when this <xref:System.Threading.Tasks.ValueTask%601> represents a successful synchronously completed operation.
```

Comment:
```suggestion
Because a <xref:System.Threading.Tasks.ValueTask%601> can be backed by a pooled <xref:System.Threading.Tasks.Sources.IValueTaskSource%601>, it is not safe to await an arbitrary <xref:System.Threading.Tasks.ValueTask%601> multiple times. You can use the <xref:System.Threading.Tasks.ValueTask%601.Preserve> method to convert this <xref:System.Threading.Tasks.ValueTask%601> into the instance backed by a regular <xref:System.Threading.Tasks.Task%601> that is safe to await multiple times. This method is similar to <xref:System.Threading.Tasks.ValueTask%601.AsTask>, but it returns the same <xref:System.Threading.Tasks.ValueTask%601> instance when this <xref:System.Threading.Tasks.ValueTask%601> represents a successful synchronously completed operation. After calling <xref:System.Threading.Tasks.ValueTask%601.Preserve> the original <xref:System.Threading.Tasks.ValueTask%601> should never be used again.
```